pkg/scheduler/apis/config/v1beta2/conversion.go

```diff
@@ -46,6 +46,98 @@ func getPluginArgConversionScheme() *runtime.Scheme {
 	return pluginArgConversionScheme
 }
 
+func Convert_v1beta2_Plugins_To_config_Plugins(in *v1beta2.Plugins, out *config.Plugins, s conversion.Scope) error {
```

Comment:
What if we get rid of pointers?

Reply:
do you mean `type Plugins struct { QueueSort PluginSet }`? instead of `*PluginSet` in the v1beta2 API?

Reply:
correct

Reply:
Done. (See fourth commit.)